contracts/swap/src/swap.rs

```diff
@@ -172,13 +188,29 @@ pub fn handle_atomic_order_reply(
         amount: vec![new_balance.clone().into()],
     };
 
+    let swap_results_json = serde_json_wasm::to_string(&swap_results).unwrap();
+    let swap_event = Event::new("atomic_swap_execution")
```

Comment:
I do not see `Route taken` , maybe in swap_results?

Requirement - https://www.notion.so/injective/Exchange-Atomic-order-smart-contract-for-Convert-9dbf4254fca14ae0a024ad82d011228a?pvs=4#4b913c3b74454b27807a8fecb1b7e68c

Reply:
yes:

```rust
pub struct SwapResults {
    pub market_id: MarketId,
    pub quantity: FPDecimal,
    pub price: FPDecimal,
    pub fee: FPDecimal,
}
```